src/hooks/useBasePopoverReactionList/types.ts

```diff
@@ -0,0 +1,65 @@
+import type {OnyxEntry} from 'react-native-onyx';
+import type {LocaleContextProps} from '@components/LocaleContextProvider';
+import type {WithCurrentUserPersonalDetailsProps} from '@components/withCurrentUserPersonalDetails';
+import type {ReactionListAnchor, ReactionListEvent} from '@pages/home/ReportScreenContext';
+import type {ReportActionReactions} from '@src/types/onyx';
+
+type BasePopoverReactionListOnyxProps = {
+    /** The reactions for the report action */
+    emojiReactions: OnyxEntry<ReportActionReactions>;
+};
+
+type BasePopoverReactionListProps = {
+    /** The ID of the report action */
+    reportActionID: string;
+
+    /** The emoji name */
+    emojiName: string;
+};
+
+type BasePopoverReactionListHookProps = BasePopoverReactionListProps & {
+    /** The reactions for the report action */
+    emojiReactions: OnyxEntry<ReportActionReactions>;
+
+    /** The current user's account ID */
+    accountID: WithCurrentUserPersonalDetailsProps['currentUserPersonalDetails']['accountID'];
+
+    preferredLocale: LocaleContextProps['preferredLocale'];
+};
+
+type BasePopoverReactionListPropsWithLocalWithOnyx = WithCurrentUserPersonalDetailsProps & BasePopoverReactionListOnyxProps & BasePopoverReactionListProps;
+type BasePopoverReactionListState = {
+    /** Whether the popover is visible */
+    isPopoverVisible: boolean;
+
+    /** The horizontal and vertical position (relative to the screen) where the popover will display. */
+    popoverAnchorPosition: {
+        horizontal: number;
+        vertical: number;
+    };
+
+    /** The horizontal and vertical position (relative to the screen) where the cursor is. */
+    cursorRelativePosition: {
+        horizontal: number;
+        vertical: number;
+    };
+};
+
+type ShowReactionList = (event: ReactionListEvent | undefined, reactionListAnchor: ReactionListAnchor) => void;
+
+type InnerReactionListRefType = {
```

Comment:
Do not use `Type` suffix for types ([1.1 Naming Conventions] from ts guidelines)